sad_comparison_functions.py

```diff
@@ -48,6 +49,23 @@ def get_pred_iterative(cdf_obs, dist, *pars):
                 return np.array(abundance)
         i += 1
 
+def get_sample_multi_dists(S, dist_name, pars):
+    """Returns a random sample of length S from the designated distribution."""
+    dist = 0
+    if dist_name == 'logser': dist = sd.logser
+    elif dist_name == 'geom': dist = sd.geom
+    elif dist_name == 'zipf': dist = sd.zipf
+    if dist:
+        rand_smp = dist.rvs(pars, size = S)
+    else:
+        if dist_name == 'negbin':
+            n, p = pars
+            rand_smp = md.nbinom_lower_trunc.rvs(n, p, size = S)
+        elif dist_name == 'pln':
+            mu, sigma = pars
+            rand_smp = md.pln.rvs(mu, sigma, True, size = S)
+    return rand_smp
+
```

Comment:
Actually, what I had before is not correct and would error. This should probably work (still untested):
```python
def get_sample_multi_dists(S, dist_name, pars):
 """Returns a random sample of length S from the designated distribution."""
    dists = {'logser': sd.logser,
             'geom': sd.geom,
             'zipf': sd.zipf,
             'negbin': md.nbinom_lower_trunc,
             'pln': md.pln}
    if dist_name = 'pln':
        #Poisson log-normal takes an extra argument for truncation
        pars += True
    rand_smp = dists[dist_name].rvs(pars*, size=S)
```